src/custom/utils/paraswap.ts

```diff
@@ -86,6 +97,11 @@ export async function getPriceQuote(params: PriceQuoteParams): Promise<ParaSwapP
     return rateResult
   } else {
     // Error getting the price
-    throw rateResult
+    const priceQuote = getPriceQuoteFromError(rateResult)
+    if (priceQuote) {
+      return priceQuote
+    } else {
+      throw rateResult
```

Comment:
you need a `try/catch` here as L100 will throw. or change L100 getPriceQuoteFromError to not throw and instead just return the error